src/playback/qml/MuseScore/Playback/internal/AuxSendControl.qml

```diff
@@ -54,7 +54,7 @@ Item {
         KnobControl {
             id: audioSignalAmountKnob
 
-            radius: root.height / 2 + 2
+            radius: root.height / 2 + 1.5
```

Comment:
The main problem seems to be with handlerHeight (KnobControl.qml), which has a hardcoded size. So, we won't need this extra height, if we just use a relative value for handlerHeight:

handlerHeight: root.radius / 2

Reply:
I tried that, but it didn't work. Its still the wrong height.

I thought the problem was that even though the Knob is not a perfect circle, its boundaries are treated like one. I understood this from line 43, where `root.radius * 2` is indirectly assigned to height. I also checked this in GIMP and verified that both width and height (including the space between two knobs) are 28 pixels. So, I believe that `AuxSendControl` expands vertically to fit this. Since, the height of the FlatButton (`bypassButton`) is the same as `AuxSendControl`'s height, there is no space between the buttons

![image](https://github.com/musescore/MuseScore/assets/88136054/8b5113aa-7d33-4112-af01-4db636e2a00d)

![image](https://github.com/musescore/MuseScore/assets/88136054/39fbf3c8-ca0e-4f59-8da9-80cf666df7fc)

What do you think?
@RomanPudashkin